### PR TITLE
submit the project form

### PR DESCRIPTION
**Pull request by melissarudofsky**
This PR allows users to submit the project form. (I got a success state from Chris, but the failure state I just made for now. )

### CODE REVIEW

File: src/components/dev-hub/project-sign-up-form.js [hunk 1]
```diff
@@ -49,10 +81,10 @@ const Form = React.memo(() => {
...
             <TextArea
-                value={project}
+                value={projectDescription}
                 required
                 placeholder="Project Description"
-                onChange={e => setProject(e.target.value)}
+                onChange={e => setProjectDescription(e.target.value)}
             />
             <SubmitContainer>
                 <Button type="submit" primary>
```
Comment (greggb) [inline src/components/dev-hub/project-sign-up-form.js]: Should we disable this onsubmit to prevent multiple POSTs

Comment (melissarudofsky) [reply src/components/dev-hub/project-sign-up-form.js]: yes!

**Comment by melissarudofsky** [comment]
added a disable state for the buttons, we probably need to expand on it for other uses but should do the trick for now!

**Comment by greggb** [comment]
Thanks

**Pull request merged**